scripts/vendor-boringssl.sh

```diff
@@ -0,0 +1,170 @@
+#!/bin/sh
+##===----------------------------------------------------------------------===##
+##
+## This source file is part of the SwiftNIO open source project
+##
+## Copyright (c) 2018 Apple Inc. and the SwiftNIO project authors
+## Licensed under Apache License v2.0
+##
+## See LICENSE.txt for license information
+## See CONTRIBUTORS.txt for the list of SwiftNIO project authors
+##
+## SPDX-License-Identifier: Apache-2.0
+##
+##===----------------------------------------------------------------------===##
+#
+# Copyright 2016, gRPC Authors All rights reserved.
```

Comment:
- is this the right way of dealing with the license headers? In `swift-nio` we have exceptions to the copyright headers in `sanity.sh`
- needs `NOTICE.txt` entry